### PR TITLE
Details coherence

### PR DESCRIPTION
**Pull request by laem**
Relates to issue #61.
Identical to the 11 last commits of #63, but working on a different html base, without the test 9731956.

### CODE REVIEW

File: assets/embauche.html [hunk 1]
```diff
@@ -261,6 +265,10 @@
 				<td><span data-source="allegement_fillon"></span>&nbsp;€</td>
 			</tr>
 			<tr>
+				<th>CICE</th>
```
Comment (MattiSG) [inline assets/embauche.html]: Why not use the full name, since we expand FNAL and CET in the same changeset?

**Comment by MattiSG** [comment]
I'd appreciate expanding the name now as I'm not sure CICE is well understood and the PR expands some other acronyms, but that's GTM, do what feels the best to you  :wink:

**Comment by MattiSG** [comment]
BTW, your detailed commit messages are :+1: :100: :smile:

File: assets/embauche.html (hunk 1 above)
Comment (laem) [inline assets/embauche.html]: :+1:

**Pull request merged**